Framework/src/CheckRunner.cxx

```diff
@@ -139,12 +139,13 @@ CheckRunner::CheckRunner(std::vector<Check> checks, std::string configurationSou
     mTotalNumberQOStored(0),
     mTotalNumberMOStored(0)
 {
+  ILOG.setFacility("Check");
   try {
     mConfigFile = ConfigurationFactory::getConfiguration(configurationSource);
   } catch (...) {
     // catch the exceptions and print it (the ultimate caller might not know how to display it)
-    ILOG(Fatal) << "Unexpected exception during initialization:\n"
-                << boost::current_exception_diagnostic_information(true) << ENDM;
+    ILOG << LogFatalSupport << "Unexpected exception during initialization:\n"
```

Comment:
I see that here exceptions are logged as fatal, while in Checks they are just errors (however, also fatal in fact). Is there any reason for this difference?

Reply:
not really, I will try to review the ones that need fatal.

Reply:
I will also send them to the Operator. If something does not start or simply stops, they should know.